webots_ros2_desktop/setup.py

```diff
@@ -1,12 +1,35 @@
 """webots_ros2 package setup file."""
 
 import os
+import shutil
 import sys
+import tarfile
+import urllib.request
 
 from setuptools import setup
 
 package_name = 'webots_ros2_desktop'
 data_files = []
+# If 'WEBOTS_HOME' not set try to download latest package (only on linux)
+if 'WEBOTS_HOME' not in os.environ and 'TRAVIS' not in os.environ and sys.platform == 'linux':
+    # Remove previous archive
+    archiveName = 'webots-R2019b-x86-64.tar.bz2'
```

Comment:
It may be better to put the Webots version in a config file instead of hardcoding it here and there.